_data/people_list.yml

```
@@ -0,0 +1,618 @@
+  email: ramesh@mathematik.uni-kl.de
+  github: Lax202
+  status: active
+
+- name: Yue Ren
+  affiliation: Durham University
+  website: https://www.yueren.de/
+  email: yue.ren2@durham.ac.uk
+  github: YueRen
+  status: active
+
+- name: Morgan Rodgers
+  affiliation: University of Kaiserslautern-Landau
+  website: https://math.rptu.de/ags/agag/personen/mitglieder
+  email: morgan.rodgers@rptu.de
+  github: mjrodgers
+  status: active
+
+- name: Felix Röhrich
+  affiliation: RWTH Aachen University
+  website: https://www.art.rwth-aachen.de/cms/~xlgua/
+  email: roehrich@art.rwth-aachen.de
+  github: felix-roehrich
+  status: active
+
+- name: Marcel Salmon
+  affiliation: Universität Oldenburg
+  github: Syz-MS
+  status: active
+  comment: Associated with Anne Fruhbig Kruger as of 2024
+
+- name: Sreevatsa Sasisekar
+  github: Sreevatsa2k
+  email: sasiseka@mathematik.uni-kl.de
+  affiliation: University of Kaiserslautern-Landau
+  status: active
+  comment: Master student of Claus Fieker as of 2024
+
+- name: Björn Schäfer
+  affiliation: Saarland University
+  email: bschaefer@math.uni-sb.de
+  website: https://www.uni-saarland.de/lehrstuhl/weber-moritz/team.html
+  status: active
+  github: BjSchaefer
+
+- name: Julien Schanz
+  affiliation: Saarland University
+  website: https://www.uni-saarland.de/lehrstuhl/weber-moritz/team/julien-schanz.html
+  email: schanz@math.uni-sb.de
+  github: julien-schanz
+  status: active
+
+- name: Johannes Schmitt
+  affiliation: Ruhr-Universität Bochum
+  website: https://joschmitt.eu
+  email: johannes.schmitt@ruhr-uni-bochum.de
+  github: joschmitt
+  paid_by_dfg: false
+  status: active
+
+- name: Hans Schönemann
+  affiliation: University of Kaiserslautern-Landau
+  website: https://math.rptu.de/ags/agag/personen/mitglieder
+  email: hans.schoenemann@math.rptu.de
+  github: hannes14
+  status: active
+
+- name: Benjamin Schröter
+  affiliation: KTH Stochkholm
+  website: https://people.kth.se/~schrot
+  email: schrot@kth.se
+  github: bschroter
+  status: active
+
+- name: Justus Springer
+  affiliation: Universität Tübingen
+  website: https://www.math.uni-tuebingen.de/de/forschung/algebra/personen/justus-springer
+  email: justus.springer@uni-tuebingen.de
+  github: justus-springer
+  status: active
+
+- name: Jan Philipp Thiele
+  affiliation: WIAS Berlin
+  github: jpthiele
+  website: https://wias-berlin.de/~thiele/
+  email: janphilipp.thiele@wias-berlin.de
+  status: active
+
+- name: Andrew P. Turner
+  affiliation: University of Pennsylvania
+  website: https://apturner.net/
+  email: turnerap@sas.upenn.edu
+  github: apturner
+  status: active
+
+- name: Grégory Vanuxem
+  github: gvanuxem
+  status: active
+  comment: Who ?
+
+- name: Laura Voggesberger
+  affiliation: Ruhr-Universität Bochum
+  website: https://www.ruhr-uni-bochum.de/ffm/Lehrstuehle/Lehrstuhl-VI/voggesberger.html
+  email: laura.voggesberger@ruhr-uni-bochum.de
+  github: voggesbe
+  status: active
```

```suggestion
- name: Laura Voggesberger
  affiliation: Ruhr-Universität Bochum
  github: voggesbe
  status: retired
```
Laura is not in Bochum anymore and left academia.